benchmark/es/string-repeat.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const assert = require('assert');
+const common = require('../common.js');
+
+const configs = {
+  n: [1e3],
+  mode: ['Array', 'repeat'],
+  encoding: ['ascii', 'utf8'],
+  size: [1e1, 1e3, 1e6],
+};
+
+const bench = common.createBenchmark(main, configs);
+
+function main(conf) {
+  const n = +conf.n;
+  const size = +conf.size;
+  const character = conf.encoding === 'ascii' ? 'a' : '\ud83d\udc0e'; // '🐎'
+
+  let str;
+
+  if (conf.mode === 'Array') {
+    bench.start();
+    for (let i = 0; i < n; i++) str = new Array(size + 1).join(character);
```

Comment:
I will wrap to conform.

It seems `let` in loops is mostly optimized now. At least they are [rather common in tests or benchmarks](https://github.com/nodejs/node/search?utf8=%E2%9C%93&q=%22for+%28let%22&type=).